config/config.go

```diff
@@ -1496,6 +1496,11 @@ type RemoteReadConfig struct {
 	// values arbitrarily into the overflow maps of further-down types.
 	HTTPClientConfig HTTPClientConfig `yaml:",inline"`
 
+	// Filter is an optional list of equality matcher names and values which
+	// expressions have to include in order to be sent to the remote read
+	// endpoint.
+	Filter model.LabelSet `yaml:"filter,omitempty"`
```

Comment:
`filter` sounds so generic and people might think it applies to returned data and not to what remote read configs even get queried.

I haven't come up with a great alternative name yet though. Something like `query_when_matches_labels`, but much shorter :D

Reply:
Select this remote read endpoint when the expression has the given labels: `selector`?

Reply:
`condition`, `matchers`, `required_matchers`

Reply:
They're not arbitrary matchers, so maybe `required_labels` ?

Reply:
`required_labels` or `required_query_labels` (probably too long) sounds good.

Reply:
Changed to `required_labels`.